src/top-bar/selection/use-selection.js

```diff
@@ -0,0 +1,4 @@
+import { useContext } from 'react'
+import { SelectionContext } from './selection-context.js'
+
+export const useSelection = () => useContext(SelectionContext)
```

Comment:
See comment in org unit select:

```suggestion
export const useSelectionContext = () => useContext(SelectionContext)
```